PokerGame/PokerGame.sln

```diff
@@ -3,18 +3,30 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
 VisualStudioVersion = 14.0.25420.1
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PokerGame", "PokerGame\PokerGame.csproj", "{B59CA04C-7D33-4420-A7C1-9F6801DEAD77}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PokerGame", "..\..\..\Users\harve\Documents\Visual Studio 2015\Projects\PokerGame\PokerGame\PokerGame.csproj", "{86B07A4C-07EF-4576-8619-2C798AF6F4B2}"
```

Comment:
You've somehow managed to include an absolute path only available on your hard drive, this project doesn't load for me because of this.  Top tip: edit this file in <insert simple text editor of your choice> to change the path back to a relative one.

Reply:
okay i think ive done so :) will commit in a bit